tests/acceptance/features/apiSettings/notificationSetting.feature

```diff
@@ -607,3 +607,24 @@ Feature: Notification Settings
       | message                                   |
       | Alice Hansen added you to Space new-space |
     But user "Brian" should not have a notification related to space "new-space" with subject "Space disabled"
+
+
+  Scenario: check share expired in-app and mail notification for Personal space resource
+    Given user "Alice" has uploaded file with content "hello world" to "testfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource           | testfile.txt         |
+      | space              | Personal             |
+      | sharee             | Brian                |
+      | shareType          | user                 |
+      | permissionsRole    | Viewer               |
+      | expirationDateTime | 2025-07-15T14:00:00Z |
+    When user "Alice" expires the last created share:
+      | space    | Personal     |
+      | resource | testfile.txt |
+    Then the HTTP status code should be "200"
+    And for user "Brian" the space "Shares" should not contain these entries:
+      | testfile.txt |
+    And user "Brian" should get a notification with subject "Membership expired" and message:
+      | message                           |
+      | Access to Space Alice Hansen lost |
```

Comment:
hmm, is this message correct? :thinking: 
Is it the same for space share?

Reply:
yes check this bug https://github.com/owncloud/ocis/issues/10936

Reply:
The issue is not related. I would expect a notification text somewhat like:
![Screenshot from 2025-02-05 15-45-54](https://github.com/user-attachments/assets/e1c2d527-5d09-4ffb-8ece-3cb6c3428cd2)
Please, open a new issue regarding the notification text and confirm if that's expected or not

Reply:
![Screenshot from 2025-02-05 16-00-57](https://github.com/user-attachments/assets/483e8746-4f3f-4b1d-82c7-795df66bcb17)

second one looks right

Reply:
https://github.com/owncloud/ocis/issues/10966

Reply:
yes, I get it. issue #10966 is more related to it than #10936